nautobot_data_validation_engine/filters.py

```diff
@@ -144,3 +146,31 @@ class Meta:
             "field",
             "error_message",
         ]
+
+
+class AuditRuleFilterSet(NautobotFilterSet):
+    """Base filterset for AuditRule model."""
+
+    q = SearchFilter(
+        filter_predicates={
+            "class_name": "icontains",
+            "method_name": "icontains",
+            "message": "icontains",
+            "content_type__app_label": "icontains",
+            "content_type__model": "icontains",
+            "object_id": "icontains",
+        }
+    )
+    class_name = filters.CharFilter(field_name="class_name", lookup_expr="icontains")
+    method_name = filters.CharFilter(field_name="method_name", lookup_expr="icontains")
+    validated_attribute = filters.CharFilter(field_name="validated_attribute", lookup_expr="icontains")
```

Comment:
These likely need not be explicitly defined. By doing so and especially with a defined `lookup_expr` you are denying the built in functionality to auto create filters for things like "starts with", "ends with", "negation" etc, plus the ability to specify multiple values. Because these are just char fields, they should just be included in `Meta.fields`.